LibraryManagementSystem/Forms/DeleteTab.cs

```diff
@@ -31,17 +31,65 @@ private void btnBack_Click(object sender, EventArgs e)
 
         private void btnDelete_Click(object sender, EventArgs e)
         {
-            con = new SqlConnection("Data Source=DESKTOP-9MBNT14\\SQLEXPRESS;Initial Catalog=libraryData;Integrated Security=True");
-            con.Open();
-            int bookID = Convert.ToInt32(txtBookID.Text);
+            if (String.IsNullOrEmpty(txtBookID.Text))
```

Comment:
Add comments to explain what's being checked on if statements and other important details